portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,35 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
     <script src="script.js"></script>
+    <style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+    }
```

Comment:
spaces after "body" and "center"

btw, would be good practice to have any common styles like these in a separate file that's imported. The reason being that if you (or later a teammate working in the same repository) want to make an adjustment, you only have to do it once instead of searching for all instances of it